src/main/java/org/jboss/arquillian/testcontainers/TestContainersObserver.java

```diff
@@ -42,7 +42,14 @@ public void createContainer(@Observes(precedence = 500) BeforeClass beforeClass)
         final DockerRequired dockerRequired = javaClass.getAnnotation(DockerRequired.class);
         if (dockerRequired != null) {
             if (!isDockerAvailable()) {
-                throw createException(dockerRequired.value());
+                // TODO Discuss non-WildFly options
+                if ("true".equals(System.getProperty("org.wildfly.test.require.docker")) ||
+                        "true".equals(System.getProperty("arquillian.testcontainers.require.docker"))) {
```

Comment:
This needs to ignore case.

Reply:
(The same bug is in WF code...)

Reply:
+same issue above.

Reply:
FWIW if we do the boolean type, I personally prefer a patter of something like:
```java
final var value = System.getProperty("org.arquillian.testcontainers.require.docker.override");
if (value != null && (value.isBlank() || "true".equalsIgnoreCase(value)) {
    throw new AssertionError("Docker is required, but it is not available or property configured");
} else {
    throw createException(dockerRequired.value());
}
```

That said, I do think I prefer allowing the exception type itself to be overridden.